0003-interledger-protocol/0003-interledger-protocol.md

```diff
@@ -245,8 +245,49 @@ Amount in discrete units of the receiving ledger's asset type. Note that the amo
 
 Arbitrary data that is attached to the payment. The contents are defined by the transport layer protocol.
 
+### ILP Rejection Format
+
+Here is a summary of the fields in the ILP error format:
+
+| Field | Type | Short Description |
+|:--|:--|:--|
+| code | IA5String | [ILP Error Code](#ilp-error-codes) |
+| triggeredBy | Address | ILP address of the entity that originally emitted the error |
+| message | UTF8String | Error data provided for debugging purposes |
+| data | OCTET STRING | Error data provided for debugging purposes |
+
+#### code
+
+    IA5String (SIZE(3))
+
+Error code. For example, `F00`. See [ILP Error Codes](#ilp-error-codes) for the list of error codes and their meanings.
+
+#### triggeredBy
+
+[ILP Address](#account) of the entity that originally emitted the error.
+
+#### message
+
+    UTF8String (SIZE(0..8191))
+
+Human-readable error message. When emitting a rejection, implementations (receivers or connectors) MAY include additional debug information. When including additional debug information, implementations SHOULD use the format `Error message. key1=value1 key2=value2`, e.g.:
+
+```
+Invalid destination. destination=example.us.bob
+```
+
+This field MUST be encoded as UTF-8.
+
+#### data
+
+    OCTET STRING (SIZE(0..32767))
+
+Machine-readable data. The format is defined for each error code. Implementations MUST follow the correct format for the code given in the `code` field.
```

Comment:
Where are each code's data formats defined? I don't see any mention of it at https://github.com/interledger/rfcs/blob/cecf8d2f6b50556bbb28427caac7ee29c6ed97f1/0003-interledger-protocol/0003-interledger-protocol.md#ilp-error-codes.

Reply:
I remembered that @emschwartz had already added a column for that in #318, so I'm planning to copy it from there. Will add that before merging. Note that currently all errors have empty data.